tensorflow/compiler/mlir/lite/experimental/tac/py_wrapper/BUILD

```diff
@@ -97,7 +101,7 @@ pybind_extension(
         "@upb//:__subpackages__",
         "@XNNPACK//:__subpackages__",
         "@zlib//:__subpackages__",
-    ],
+    ] + if_onednn_v3(["@onednn_v3//:__subpackages__"]),
```

Comment:
Do you know what this target need oneDNN for?

Reply:
I initially added it because this target uses `@mkl_dnn_v1`, but seems like it is not really needed as build passes for both oneDNN v2.x and v3.x even without `@onednn_v3`. So I've removed this dependency here and in all other places where it is not really needed. 

Thanks for catching this!

Reply:
Thank you for checking! The commit removing this line wasn't pulled in when we merged this PR. I'll just remove it later.

Reply:
Sounds good, thanks!